### PR TITLE
LRAC-13369 | master

### PR DESCRIPTION
**Pull request by leilanyaragao**
Jira Issue: https://issues.liferay.com/browse/LRAC-13369

### CODE REVIEW

**Comment by leilanyaragao** [comment]
ci:test:sf

File: modules/dxp/apps/osb/osb-faro/osb-faro-contacts-service/service.xml [hunk 1]
```diff
@@ -16,9 +16,10 @@
 
 		<!-- Audit fields -->
 
+		<column name="companyId" type="long" />
```
Comment (marcosapmf) [inline modules/dxp/apps/osb/osb-faro/osb-faro-contacts-service/service.xml:19]: Please see https://github.com/brianchandotcom/liferay-portal/pull/133577#issuecomment-1514143081

File: modules/dxp/apps/osb/osb-faro/osb-faro-contacts-service/service.xml [hunk 2]
```diff
@@ -45,9 +46,10 @@
...
```
Comment (marcosapmf) [inline modules/dxp/apps/osb/osb-faro/osb-faro-contacts-service/service.xml:49]: The same as above

File: modules/dxp/apps/osb/osb-faro/osb-faro-contacts-service/src/main/java/com/liferay/osb/faro/contacts/service/base/ContactsCardTemplateLocalServiceBaseImpl.java [hunk 3]
```diff
@@ -67,7 +69,8 @@
  */
 public abstract class ContactsCardTemplateLocalServiceBaseImpl
 	extends BaseLocalServiceImpl
-	implements ContactsCardTemplateLocalService, IdentifiableOSGiService {
+	implements AopService, ContactsCardTemplateLocalService,
```
Comment (marcosapmf) [inline modules/dxp/apps/osb/osb-faro/osb-faro-contacts-service/src/main/java/com/liferay/osb/faro/contacts/service/base/ContactsCardTemplateLocalServiceBaseImpl.java:72]: *BaseImpl makes part of `BuildService`

File: modules/dxp/apps/osb/osb-faro/osb-faro-contacts-service/src/main/java/com/liferay/osb/faro/contacts/service/base/ContactsLayoutTemplateLocalServiceBaseImpl.java [hunk 4]
```diff
@@ -67,7 +69,8 @@
  */
 public abstract class ContactsLayoutTemplateLocalServiceBaseImpl
```
Comment (marcosapmf) [inline modules/dxp/apps/osb/osb-faro/osb-faro-contacts-service/src/main/java/com/liferay/osb/faro/contacts/service/base/ContactsLayoutTemplateLocalServiceBaseImpl.java:70]: *BaseImpl makes part of BuildService

**Review by marcosapmf — CHANGES_REQUESTED** [review]

**Comment by marcosapmf** [comment]
@leilanyaragao please unify the following commits:
[LRAC-13369 osb-faro-contacts-service:BuildService](https://github.com/liferay-ac/liferay-portal/pull/371/commits/32a8bc979f7977997182b2075b05859f2d8c3a29)
[LRAC-13369 osb-faro-contacts-api:BuildService](https://github.com/liferay-ac/liferay-portal/pull/371/commits/87a78b5f2ff149a92b8666fca17a65983aa3907c)
[LRAC-13369 osb-faro-contacts-service:Apply](https://github.com/liferay-ac/liferay-portal/pull/371/commits/08b9a871955998252d9567f443f5419ab776b82c)

**Comment by leilanyaragao** [comment]
ci:test:sf

**Comment by leilanyaragao** [comment]
ci:test:stable

**Comment by marcosapmf** [comment]
ci:test:stable

**Comment by leilanyaragao** [comment]
ci:test:stable

**Comment by leilanyaragao** [comment]
ci:test:sf

**Pull request closed without merging**